osbs/api.py

```diff
@@ -505,7 +505,8 @@ def _do_create_prod_build(self, git_uri, git_ref,
             info_url_format=self.build_conf.get_info_url_format(),
             artifacts_allowed_domains=self.build_conf.get_artifacts_allowed_domains(),
             low_priority_node_selector=self.build_conf.get_low_priority_node_selector(),
-            equal_labels=self.build_conf.get_equal_labels()
+            equal_labels=self.build_conf.get_equal_labels(),
+            node_selector_platforms=self.build_conf.get_node_selector_platform()
```

Comment:
We might want to do this inside `create_worker_build()`, and pass through the node selector for the specified platform.